code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -19,15 +19,14 @@
 	mob_name = "a swarmer"
 	death = FALSE
 	roundstart = FALSE
-	flavour_text = {"
-	<b>You are a swarmer, a weapon of a long dead civilization. Until further orders from your original masters are received, you must continue to consume and replicate.</b>
-	<b>Clicking on any object will try to consume it, either deconstructing it into its components, destroying it, or integrating any materials it has into you if successful.</b>
-	<b>Ctrl-Clicking on a mob will attempt to remove it from the area and place it in a safe environment for storage.</b>
-	<b>Objectives:</b>
+	important_info = "Follow your objectives, do not make the station inhospitable or try and kill crew."
```

Comment:
Can you add a comment to show where each of these will appear, like mention if important_info is the bold red one thats for OOC rules, etc